install/includes/manage-services.rst

```diff
@@ -9,10 +9,10 @@ with the parent ``zammad``.
    $ # Zammad service to start all services at once
    $ systemctl (status|start|stop|restart) zammad
 
-   $ # Zammads internal railsserver
+   $ # Zammads internal puma server (relevant for display the web app)
```

Comment:
"displaying" instead of "display"? Not 100% sure though.

Reply:
Yes you're right. 🙈